src/components/edit-movie/edit-movie.module.scss

```diff
@@ -0,0 +1,83 @@
+@import '../../palette.scss';
+
+.content {
+  display: flex;
+  justify-content: space-between;
+  width: 859px;
+}
+
+.title {
+  display: flex;
+  justify-self: flex-start;
+  font-weight: 300;
+  font-size: 40px;
+  line-height: 49px;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  margin-bottom: 36px;
+}
+
+.textWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.label {
+  margin-bottom: 13px;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 20px;
+  letter-spacing: 0.888889px;
+  text-transform: uppercase;
+  color: $red;
+  opacity: 0.8;
+}
+
+.firstCol {
+  width: 490px;
+  height: 57px;
+}
+.secondCol {
```

Comment:
Try to separate your CSS styles with empty lines, please. It becomes really hard to read and pay attention for pseudo classes, elements with code overlapping each other